.github/workflows/validate.yml

```diff
@@ -0,0 +1,20 @@
+name: JSON Schema Validation
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Download the JSON schema
+        run: wget https://raw.githubusercontent.com/ossf/osv-schema/v1.6.0/validation/schema.json
+
+      - name: Install check-jsonschema
+        run: pip install check-jsonschema
+
+      - name: Validate JSON
+        run: check-jsonschema --schemafile schema.json upstream/*.json
```

Comment:
Do we need to check all `*.json` files inside of `upstream` folder when checking PR?
Perhaps we only want to check new files.

Reply:
How is this change?
https://github.com/aquasecurity/vuln-list-k8s/pull/95/commits/f03011638b9dde9eb0a76c6563af72fa37471f28

Reply:
Perfect solution.
This is even better than using other actions from marketplace.